src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -616,7 +616,9 @@ protected function selectTableColumns(string $databaseName, ?string $tableName =
             $sql .= ' c.relname AS table_name, n.nspname AS schema_name,';
         }
 
-        $sql .= <<<'SQL'
+        $fragmentDefaultValueSQL = $this->_platform->getDefaultColumnValueSQLSnippet();
```

Comment:
Up for debate, but I think I'd rather resort to `sprintf` here, partly because it means no extra variable is required.

Reply:
I agree to disagree, if you can do a suggestion I will gladly accept it